src/inverseRobot.js

```diff
@@ -23,8 +23,25 @@
  *
  * @return {object}
  */
-function inverseRobot(robot) {
-  // write code here
+function inverseRobot(robot = {}) {
+  const inputRobot = { ...robot };
+  const reverseRobot = {};
+  const organs = [];
+
+  for (const elem of Object.entries(inputRobot)) {
+    organs.push(elem[1]);
+    reverseRobot[elem[1]] = elem[0];
+  }
+
+  for (const item of organs) {
+    if (organs.filter(organ => organ === item).length > 1) {
+      return null;
+    }
+
+    return reverseRobot;
+  }
+
+  return inputRobot;
```

Comment:
Looks like last return is never happened. 
Why do you use it? We don't need to return input robot according to task conditions

Reply:
No, whitout this return everything is falling apart. 
When everything okey with robot, we must return clear input